habit/.prettierrc.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
-  bracketSpacing: false,
+  bracketSpacing: true,
```

Comment:
what is this module?

Reply:
Install prettier, helps maintain code neat and clean